Source/Internal/Handle/DrivingScrollViewViewModifier.swift

```diff
@@ -13,6 +13,35 @@ struct DrivingScrollViewViewModifier: ViewModifier {
     let isActive: Bool
 
     func body(content: Content) -> some View {
-        content.preference(key: DynamicOverlayScrollPreferenceKey.self, value: isActive)
+        DrivingScrollViewWrapper(content: content)
+            .preference(key: DynamicOverlayScrollPreferenceKey.self,
+                        value: isActive)
+    }
+}
+
+struct DrivingScrollViewWrapper<Content: View>: UIViewRepresentable {
```

Comment:
Could you use a `UIViewControllerRepresentable` instead?

I think it is not a good idea to create a `UIHostingController` and use only its view.

Reply:
I have changed to using a UIViewControllerRepresentable and instead of using the MarkingWrapper the detection is done, by setting the accessiblityIdentifier, since I could not find a way how to use the MarkingWrapper still. To prevent  maybe changing the identifier of the users content I wrap it in a ZStack.